.orchestra/support/revng-container/install.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+set -euo pipefail
+
+cd "$BUILD_DIR"
+UUID="$(uuidgen)"
+IMAGE_NAME="revng-container_$UUID"
+TEST_NAME=""
+
+function cleanup {
+  trap - EXIT;
+  if [ -n "${TEST_NAME-}" ]; then
+    podman kill "$TEST_NAME"
+    podman rm -f "$TEST_NAME"
+  fi
+  podman rmi -i "$IMAGE_NAME"
+}
+trap cleanup EXIT
+
+
+rsync \
+  --archive \
+  --verbose \
+  --progress \
+  "$ORCHESTRA_ROOT/revng" ./
+
+cp -a \
+  "$ORCHESTRA_DOTDIR/support/revng-container/"{apt.no-extra,entrypoint.sh,requirements.txt} \
+  ./
+
+podman build \
+  -t "$IMAGE_NAME" \
+  -f "$ORCHESTRA_DOTDIR/support/revng-container/Dockerfile" \
+  .
+
+podman save "$IMAGE_NAME" | xz -cv3 -T0 > "${TMP_ROOT}${ORCHESTRA_ROOT}/share/revng-container.tar.xz"
+
+if [ "$RUN_TESTS" -eq 1 ]; then
+  TEST_NAME="revng-container-test_$UUID"
+  TEST_PORT="$((1024 + RANDOM))"
```

Comment:
This could produce a number greater than 2**16, I think.

Reply:
Fixed via the use of unix sockets